RFC/src/RFC-0201_TariScript.md

```diff
@@ -451,90 +562,96 @@ validation steps.
 ### Standard MW transaction
 
 For this use case we have Alice who sends Bob some Tari.
-Bob's wallet is online and is able to countersign the transaction.
+Bob's wallet is  online and is able to countersign the transaction.
 
-Alice creates a new transaction spending \\( C_a \\) to a new output containing the commitment \\( C_b \\) (ignoring fees for now).
+Alice creates a new transaction spending \\( C\_a \\) to a new output containing the commitment \\( C\_b \\) (ignoring fees for now).
 
-To spend \\( C_a \\), she provides:
+To spend \\( C\_a \\), she provides:
 
-- An input that contains \\( C_a \\).
-- The script input, \\( \input_a \\).
-- A valid script signature, \\( (a*{Sa}, b*{Sa}, R*{Sa}) \\) as per (3),(4) proving that she owns the commitment
-  \\( C_a \\), knows the private key, \\( k*{Sa} \\), corresponding to \\( K\_{Sa} \\), the public key left on the stack
+* An input that contains \\( C\_a \\).
+* The script input, \\( \input_a \\).
+* A valid script signature, \\( (a_{Sa}, b_{Sa}, R_{Sa}) \\) as per (13),(14) proving that she owns the commitment 
```

Comment:
Here we talk about the tuple of the signature again... We should be consistent. My vote is in favor of a single value.